test/e2e/validation_test.go

```diff
@@ -96,26 +96,7 @@ func (tc *testContext) getWindowsVM(ipAddress, instanceID string) (types.Windows
 	if creds == (wmc.Credentials{}) {
 		return nil, errors.New("expected credentials to be present but got a nil value")
 	}
-	winVM, err := wmc.GetWindowsVM(instanceID, ipAddress, creds)
-	return winVM, nil
-}
-
-// validateConnectivity ensures that we have connectivity for the Windows VM
-func (tc *testContext) validateConnectivity(windowsVM types.WindowsVM) error {
```

Comment:
This test anyways needs to be taken out, this was written as an example for others to use.